### PR TITLE
Feat/more viso

### LINKED ISSUE #70: 2d: Visualize the distribution of trip lengths per month
Visualize the distribution of trip lengths per month. Compare the distributions to normal
distributions with mean and standard deviation as calculated before (1.d))

### LINKED ISSUE #72: 3b: Try other models
Use other classification algorithms than KNN

### LINKED ISSUE #80: Rename column names in concise way
Use Starting upper case latter and each time "_" instead of " " between words

### PR DESCRIPTION
**Pull request by weisske**
This merge should be done after the merge of docstring branch

### CODE REVIEW

**Comment by weisske** [comment]
sorry for prediction in viso branch! But I was in a huge workflow :-D

File: nextbike/prediction/geo_predict.py [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=42)
-    clf = KNeighborsClassifier()
+    X_scaled = StandardScaler().fit_transform(X)
+    X_train, X_test, y_train, y_test = train_test_split(X_scaled, y, test_size=0.3, random_state=42)
+    # train
+    # KNN
+    clf = KNeighborsClassifier(n_neighbors=20, weights="distance")
     clf.fit(X_train, y_train)
-    print("Test score (mean accuracy)", np.round(clf.score(X_test, y_test)*100, 2), "%")
+    # Decision Tree
```
Comment (Speedfan94) [inline nextbike/prediction/geo_predict.py:31]: What is done here?

File: nextbike/visualization/math_descriptive.py [hunk 2]
```diff
@@ -170,70 +250,7 @@ def plot_mean_duration(p_df):
         p_sub_folder1="data_plots",
         p_sub_folder2="math"
     )
-
```
Comment (Speedfan94) [inline nextbike/visualization/math_descriptive.py:173]: why did you delete this?

**Review by Speedfan94 — COMMENTED** [review]
some comments

File: nextbike/prediction/geo_predict.py (hunk 1 above)
Comment (weisske) [reply nextbike/prediction/geo_predict.py:31]: add a comment and drop another line :-P

File: nextbike/visualization/math_descriptive.py (hunk 2 above)
Comment (weisske) [reply nextbike/visualization/math_descriptive.py:173]: because it is an empty line ;-)

Comment (Speedfan94) [reply nextbike/visualization/math_descriptive.py:173]: moved to new math_preditive.py

**Review by Speedfan94 — APPROVED** [review]

**Review by mulko93 — APPROVED** [review]
these changes can be merged into dev (team review)

File: nextbike/visualization/geo.py [hunk 3]
```diff
@@ -104,32 +104,36 @@ def visualize_plz(p_df):
     # Visualizes the number of started trip for each zip code region for the month with the most trips
 
     # find the month with the most trips:
-    p_df["month"] = p_df["Start Time"].dt.month
+    p_df["Month"] = p_df["Start_Time"].dt.month
```
Comment (MaxStroh) [inline nextbike/visualization/geo.py:107]: We should have Month_start here already - we might not need this anymore :)

**Review by MaxStroh — APPROVED** [review]
Looks fine :)

File: nextbike/visualization/geo.py (hunk 3 above)
Comment (MaxStroh) [reply nextbike/visualization/geo.py:107]: Discussed in live session. We will check this after the merge

**Comment by MaxStroh** [comment]
Reviewed and discussed in live session
